### PR TITLE
Kernel: Merge the x86 and x86_64 boot code into a single file

### PR DESCRIPTION
**Pull request by gunnarbeutner**
They're mostly the same apart from some x86_64-specific parts.

### CODE REVIEW

**Comment by Hendiadyoin1** [comment]
Good changes, you can include `Kernel/Section.h` too, to get the Kernel Base from the place the rest of the Kernel takes it from.

**Comment by gunnarbeutner** [comment]
> Good changes, you can include `Kernel/Section.h` too, to get the Kernel Base from the place the rest of the Kernel takes it from.

Aye, good idea. I've added a commit that does just that. :)

File: Kernel/Sections.h [hunk 1]
```diff
@@ -11,4 +11,4 @@
 #define READONLY_AFTER_INIT __attribute__((section(".ro_after_init")))
 #define UNMAP_AFTER_INIT NEVER_INLINE __attribute__((section(".unmap_after_init")))
 
-#define KERNEL_BASE 0xC000'0000
```
Comment (Hendiadyoin1) [inline Kernel/Sections.h:14]: oh yeah, apperantly assembly does not like seperators in numbers...
my bad

**Review by awesomekling — APPROVED** [review]
LGTM but please fix CI sadness

**Pull request merged**